pkg/endpoint/directory.go

```diff
@@ -23,31 +23,20 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-func getTempEndpointDirectory(origDir string) string {
-	return origDir + "_next"
-}
-
 // synchronizeDirectories moves the files related to endpoint BPF program
 // compilation to their according directories based on whether an error occurred
 // during regeneration, or if compilation of BPF was necessary for the endpoint.
 // Returns the original regenerationError if regenerationError was non-nil,
 // or if any updates to directories for the endpoint's directories fails.
 // Must be called with endpoint.Mutex held.
-func (e *Endpoint) synchronizeDirectories(origDir string, compilationExecuted bool, regenerationError error) error {
+func (e *Endpoint) synchronizeDirectories(origDir string, compilationExecuted bool) error {
```

Comment:
Update documentation to remove references to regeneration error

Reply:
Done